packages/@okta/vuepress-site/docs/reference/api/groups/index.md

```diff
@@ -849,10 +849,9 @@ curl -v -X PUT \
 
 <ApiOperation method="delete" url="/api/v1/groups/${groupId}" />
 
-Removes a group with `OKTA_GROUP` type from your organization.
+Removes a group with `OKTA_GROUP` or `APP_GROUP` type from your organization.
 
-> Only groups with `OKTA_GROUP` type can be removed.<br>
-> Application imports are responsible for removing groups with `APP_GROUP` type such as Active Directory groups.
+> Groups with type `APP_GROUP` cannot be removed if they are used in a group push mapping.
```

Comment:
Is group push the only check we're doing when deleting the app group? Anything else we should mention here?

Reply:
To my knowledge, yes, this is the only check.

Reply:
Looks good to me then!